OpenRA.Game/VoiceExts.cs

```diff
@@ -0,0 +1,74 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA
+{
+	public static class VoiceExts
+	{
+		public static void PlayVoice(this Actor self, string phrase, string variant)
```

Comment:
We probably want to keep the variant string on the lower level code for now though, as they are used for different things by e.g. notifications.